packages/form/src/cli.ts

```diff
@@ -45,8 +101,19 @@ async function create() {
 				if (!value) {
 					return "Please enter a path.";
 				}
-				if (existsSync(resolve(process.cwd(), value, "Form.tsx"))) {
-					return "A Form.tsx component already exists here.";
+				if (
+					fileNamesToCreate.some((fileName) =>
+						existsSync(
+							resolve(
+								process.cwd(),
+								value,
+								toUseFramework.templateDir,
+								fileName,
+							),
+						),
```

Comment:
```suggestion
						existsSync(resolve(process.cwd(), value, fileName)),
```

I noticed this checks for files in `[input]/react` for example, rather than just `[input]`. May have lost something in translation there! But the intent was to copy where the user chooses without nesting the framework name